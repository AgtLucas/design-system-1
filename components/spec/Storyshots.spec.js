import initStoryshots, { multiSnapshotWithOptions } from "@storybook/addon-storyshots";
import { mount } from "enzyme";
import { createSerializer } from "enzyme-to-json";
import "jest-styled-components";

initStoryshots({
  storyNameRegex: /^((?!.*?SkipStoryshot).)*$/,
  snapshotSerializers: [createSerializer()],
  test: multiSnapshotWithOptions({
    renderer: mount
  })
});
