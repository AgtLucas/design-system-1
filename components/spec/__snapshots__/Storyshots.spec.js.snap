// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Box Box 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With a background colour 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With a responsive width 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With a set width 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With a shadow 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With a text colour 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With margin 1`] = `ReactWrapper {}`;

exports[`Storyshots Box With padding 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons As a link 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons Button 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons DangerButton 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons PrimaryButton 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons QuietButton 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons Set to full width 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons With a selected Icon 1`] = `ReactWrapper {}`;

exports[`Storyshots Buttons With a selected size 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Checkbox 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Controlled 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Set to defaultChecked 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Set to error 1`] = `ReactWrapper {}`;

exports[`Storyshots Checkbox Set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup Checkbox Group 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup Controlled 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup with additional text 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup with defaultValue 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup with helpText 1`] = `ReactWrapper {}`;

exports[`Storyshots CheckboxGroup with requirementText 1`] = `ReactWrapper {}`;

exports[`Storyshots DemoPage Demo Page 1`] = `ReactWrapper {}`;

exports[`Storyshots Field Field 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With Input 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With Input and InlineValidation 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With Label for Input 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With a long text 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With all additional components 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With help text 1`] = `ReactWrapper {}`;

exports[`Storyshots Field With requirement text 1`] = `ReactWrapper {}`;

exports[`Storyshots FieldLabel FieldLabel 1`] = `ReactWrapper {}`;

exports[`Storyshots FieldLabel with HelpText 1`] = `ReactWrapper {}`;

exports[`Storyshots FieldLabel with RequirementText 1`] = `ReactWrapper {}`;

exports[`Storyshots FieldLabel with all additional text 1`] = `ReactWrapper {}`;

exports[`Storyshots FieldLabel with associated custom input component 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex Flex 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex With custom order 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex alignItems set to center 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex alignItems set to flex-end 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex alignItems set to flex-start 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex alignItems set to stretch (default) 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexDirection set to column 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexDirection set to column-reverse 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexDirection set to row-reverse 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexWrap set to no-wrap (default) 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexWrap set to wrap 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex flexWrap set to wrap-reverse 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to center 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to flex-end 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to flex-start (default) 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to space-around 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to space-between 1`] = `ReactWrapper {}`;

exports[`Storyshots Flex justifyContent set to space-evenly 1`] = `ReactWrapper {}`;

exports[`Storyshots Form Demo form 1`] = `ReactWrapper {}`;

exports[`Storyshots Form Form 1`] = `ReactWrapper {}`;

exports[`Storyshots Form With form sections 1`] = `ReactWrapper {}`;

exports[`Storyshots Form With form sections without titles 1`] = `ReactWrapper {}`;

exports[`Storyshots Form Without title 1`] = `ReactWrapper {}`;

exports[`Storyshots Header Validation Header Validation 1`] = `ReactWrapper {}`;

exports[`Storyshots Header Validation With message only 1`] = `ReactWrapper {}`;

exports[`Storyshots Headings Section Title 1`] = `ReactWrapper {}`;

exports[`Storyshots Headings Subsection Title 1`] = `ReactWrapper {}`;

exports[`Storyshots Headings Title 1`] = `ReactWrapper {}`;

exports[`Storyshots Headings With a custom margin 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon Icon 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon InlineIcon 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon With a color 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon With a size 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon With accessibility title 1`] = `ReactWrapper {}`;

exports[`Storyshots Icon With added margin 1`] = `ReactWrapper {}`;

exports[`Storyshots IconicButton Disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots IconicButton With a long label 1`] = `ReactWrapper {}`;

exports[`Storyshots IconicButton With label 1`] = `ReactWrapper {}`;

exports[`Storyshots Inline Validation Inline Validation 1`] = `ReactWrapper {}`;

exports[`Storyshots Inline Validation With list items 1`] = `ReactWrapper {}`;

exports[`Storyshots Input Input 1`] = `ReactWrapper {}`;

exports[`Storyshots Input Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Input Set to error 1`] = `ReactWrapper {}`;

exports[`Storyshots Input Set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots Link Link 1`] = `ReactWrapper {}`;

exports[`Storyshots Link With a different color 1`] = `ReactWrapper {}`;

exports[`Storyshots Link Without underline 1`] = `ReactWrapper {}`;

exports[`Storyshots List List 1`] = `ReactWrapper {}`;

exports[`Storyshots List With compact spacing 1`] = `ReactWrapper {}`;

exports[`Storyshots List With custom colour 1`] = `ReactWrapper {}`;

exports[`Storyshots List With custom font size and line height 1`] = `ReactWrapper {}`;

exports[`Storyshots MenuDropdown MenuDropdown 1`] = `ReactWrapper {}`;

exports[`Storyshots MenuItem MenuItem 1`] = `ReactWrapper {}`;

exports[`Storyshots NavBar NavBar 1`] = `ReactWrapper {}`;

exports[`Storyshots NavBarSearch NavBarSearch 1`] = `ReactWrapper {}`;

exports[`Storyshots NavBarSearch With custom name 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Controlled 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Radio 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Set to defaultChecked 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Set to error 1`] = `ReactWrapper {}`;

exports[`Storyshots Radio Set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup Controlled 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup Radio Group 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup with additional text 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup with helpText 1`] = `ReactWrapper {}`;

exports[`Storyshots RadioGroup with requirementText 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Select 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Select as a controlled component 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Select with an option selected 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Set to error 1`] = `ReactWrapper {}`;

exports[`Storyshots Select Set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots Text Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Text Set to inline 1`] = `ReactWrapper {}`;

exports[`Storyshots Text Text 1`] = `ReactWrapper {}`;

exports[`Storyshots Text With a color 1`] = `ReactWrapper {}`;

exports[`Storyshots Text With a custom margin 1`] = `ReactWrapper {}`;

exports[`Storyshots Text With a size 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea Set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea Set to error 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea Set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea Textarea 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea With custom number of rows 1`] = `ReactWrapper {}`;

exports[`Storyshots Textarea With placeholder text 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Controlled Toggle 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Toggle 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Toggle Field 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Toggle set to defaultToggled 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Toggle set to disabled 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle Toggle set to required 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle With long text 1`] = `ReactWrapper {}`;

exports[`Storyshots Toggle With text 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip Tooltip 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with Button passed in 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with Link passed in 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with custom hideDelay 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with custom maxWidth 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with custom showDelay 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with other focusable elements 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with placement 1`] = `ReactWrapper {}`;

exports[`Storyshots Tooltip with wrapped text 1`] = `ReactWrapper {}`;

exports[`Storyshots Typography Article 1`] = `ReactWrapper {}`;
